From c68241016d2244018a1099bfb5c6b264dd5cbdc2 Mon Sep 17 00:00:00 2001
From: redfish <redfish@galactica.pw>
Date: Sat, 28 Apr 2018 16:04:07 -0400
Subject: [PATCH] cmake: do not install into the system

This fork is used a vendored dependency, and we should not install it
into the system, because it conflicts with miniupnpc installed in the
system.
---
 miniupnpc/CMakeLists.txt | 20 --------------------
 1 file changed, 20 deletions(-)

diff --git a/miniupnpc/CMakeLists.txt b/miniupnpc/CMakeLists.txt
index cb997dc..87df7ae 100644
--- a/miniupnpc/CMakeLists.txt
+++ b/miniupnpc/CMakeLists.txt
@@ -177,26 +177,6 @@ if (UPNPC_BUILD_TESTS)
 # set (UPNPC_INSTALL_TARGETS ${UPNPC_INSTALL_TARGETS} testminixml minixmlvalid testupnpreplyparse testigddescparse testminiwget)
 endif (UPNPC_BUILD_TESTS)
 
-
-install (TARGETS ${UPNPC_INSTALL_TARGETS}
-  RUNTIME DESTINATION bin
-  LIBRARY DESTINATION lib${LIB_SUFFIX}
-  ARCHIVE DESTINATION lib${LIB_SUFFIX}
-)
-install (FILES
-	miniupnpc.h
-  miniwget.h
-  upnpcommands.h
-  igd_desc_parse.h
-  upnpreplyparse.h
-  upnperrors.h
-  upnpdev.h
-  miniupnpctypes.h
-  portlistingparse.h
-  miniupnpc_declspec.h
-  DESTINATION include/miniupnpc
-)
-
 # commented out by Ben Boeckel, who I presume knows what he's doing;)
 #set (CONFIGURED YES CACHE INTERNAL "")
 
-- 
2.17.0

